dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/RecoverInterruptedJob.java

```diff
@@ -54,168 +65,180 @@
     private final EndpointInfo endpointInfo;
     private final JobInterruptionStore jobInterruptionStore;
     private final ResourceBundle resourceBundle;
-    private final boolean isCacheJobEnable;
+    private final SettingsStore settingsStore;
 
-    public RecoverInterruptedJob(final UUID uuid,
-                                 final EndpointInfo endpointInfo,
-                                 final JobInterruptionStore jobInterruptionStore,
-                                 final boolean isCacheJobEnable) {
+    @Inject
+    public RecoverInterruptedJob(@Assisted final UUID uuid, @Assisted final EndpointInfo endpointInfo, final JobInterruptionStore jobInterruptionStore, final Ds3Client client, final LoggingService loggingService, final SettingsStore settingsStore, final ResourceBundle resourceBundle) {
```

Comment:
this line is really long - personally I prefer the previous way with one param per line

Reply:
I'm with Denver on this one.